am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -219,6 +250,52 @@ public FilteredResourceEnvelope filterResource(@NotBlank String userId,
             .build();
     }
 
+    /**
+     * Removes empty nodes from data.
+     *
+     * @param node JsonNode
+     */
+    private void removeEmptyNodeValues(JsonNode node) {
+        if (nonNull(node)) {
+            Iterator<JsonNode> it = node.iterator();
```

Comment:
is this method necessary? Empty nodes should not be deleted as discussed with @shashank-rastogi

Reply:
Lets check business around that, but why we need empty nodes to be returned.

Reply:
removed this.